os/drivers/serial/serial.c

```diff
@@ -265,6 +267,9 @@ static int uart_putxmitchar(FAR uart_dev_t *dev, int ch, bool oktoblock)
 	for (;;) {
 		if (nexthead != dev->xmit.tail) {
 			dev->xmit.buffer[dev->xmit.head] = ch;
+#ifdef CONFIG_LOG_DUMP
+			add_char_to_log_dump(ch);
```

Comment:
Usually function naming is `module name` + `verb` + xxx and you did for others.
How about changing this as `log_dump_save(ch)`?